photon-client/src/main/scala/com/linkedin/photon/ml/util/DaysRange.scala

```diff
@@ -35,11 +35,12 @@ case class DaysRange(startDays: Int, endDays: Int) {
   /**
    * Convert the [[DaysRange]] to a [[DateRange]].
    *
+   * @param timeZone The local timezone to use for date math
    * @return The [[DateRange]] equivalent of the [[DaysRange]] at the time of function call
    */
-  def toDateRange: DateRange = {
+  def toDateRange(timeZone: DateTimeZone = DateTimeZone.UTC): DateRange = {
```

Comment:
Since we have `DateTimeZone.UTC` appearing so much, maybe we should move it into `Constants` and have the drivers + date classes reference it from there.

Reply:
Done.